### ISSUE TITLE
Local imports using webpack aliases are being incorrectly identified as separate groups

### ISSUE THREAD

**Issue by cheapsteak** [benmosher/eslint-plugin-import#1034]
Using a webpack config using aliases (and eslint-import-resolver-webpack), each import for an internal modules is being treated as a separate "group". With `'newlines-between': 'always-and-inside-groups'`, they each have to be separated by a new line

```js
resolve: {
  alias: {
    '~': path.resolve(__dirname, 'src'),
  },
},
```

```js
import React from 'react';

import ComponentA from '~/components/ComponentA'; // [eslint] There should be at least one empty line between import groups (import/order)
import ComponentB from '~/components/ComponentB'; // [eslint] There should be at least one empty line between import groups (import/order)
import ComponentC from '~/components/ComponentC'; // no error

import ComponentD from '~/components/ComponentD'; // no error
```

Reproduced here: https://github.com/cheapsteak/eslint-import-order-bug-reproduction
Pull locally and check the lint messages for this file - https://github.com/cheapsteak/eslint-import-order-bug-reproduction/blob/master/src/App.js

**Comment by fsmaia** [comment]
I'm having a different issue, but I think they are related.

In my case, I have some aliases set in webpack, and resolver is properly configured:
```js
resolve: {
  alias: {
    actions: path.join(__dirname, './src/actions'),
    components: path.join(__dirname, './src/components'),
    constants: path.join(__dirname, './src/constants'),
    //...
  },
},
```

And `import/order` is configured with:
```json
{
  "groups": ["builtin", "external", "internal", "parent", "sibling", "index"],
  "newlines-between": "always"
}
```

I'm trying to import a external followed by internal module:
```js
import { createSelector } from 'reselect';

import { MY_CONSTANT_EXAMPLE } from 'constants/my-constant';
```

However, I'm getting this error:
> [eslint] `constants/my-constant` import should occur before import of `reselect` (import/order)

The example above works fine if I replace `constants` with `src/constants`.

**Comment by laugri** [comment]
Hi @cheapsteak , did you find a workaround since you ran into this issue ?

**Comment by cheapsteak** [comment]
Unfortunately not

**Comment by doits** [comment]
+1 named imports should be all in the internal group

**Comment by echenley** [comment]
@fsmaia I am having the exact same issue with a `constants` folder in particular. All of my other internal aliases seem to work.

**Comment by echenley** [comment]
@ljharb It seems like `isBuiltIn` needs to be updated to check for a resolved path:

https://github.com/benmosher/eslint-plugin-import/blob/master/src/core/importType.js#L24-L28

```js
// import 'fs' - path is null
// import 'constants' - path is '/absolute/path/src/constants' with webpack resolver
export function isBuiltIn(name, settings, path) {
  const base = baseModule(name)
  const extras = (settings && settings['import/core-modules']) || []
  return !path && coreModules[base] || extras.indexOf(base) > -1
}
```

**Comment by ljharb** [comment]
@echenley a PR with a test case would be appreciated :-)

**Comment by ljharb** [comment]
Now that #1297 is merged, is there anything left to do here?

**Comment by echenley** [comment]
I believe this can be closed.
